libs/zgui/src/plot.zig

```diff
@@ -537,6 +537,32 @@ extern fn zguiPlot_PlotShaded(
     stride: i32,
 ) void;
 
+pub const DragToolFlags = packed struct(u32) {
+    no_cursors: bool = false,
+    no_fit: bool = false,
+    no_no_inputs: bool = false,
+    delayed: bool = false,
+    _padding: u28 = 0,
+};
+const DragPointArgs = struct {
```

Comment:
```suggestion
const DragPoint = struct {
```
Let's change the name for consistency.

Reply:
resolved in #332